Reviews section

rzotr2 opened the pull request:
![photo_2022-12-15_12-27-53](https://user-images.githubusercontent.com/78299696/211679047-532dda0d-01bd-4ca8-bacc-3e4c244acf69.jpg)

rzotr2 committed:
[Reviews Section]

- Added html
- Stylesheet is not ready

rzotr2 committed:
Merge remote-tracking branch 'origin/reviews-section' into reviews-section

yehor-akunishnikov committed:
[Reviews Section]

- Remove unnecessary bullshit
- Rearrange scss code
- Removed clouds bg

rzotr2 committed:
[Reviews Section]

- Finished HTML
- Finished CSS
- Section is ready to be reviewed

rzotr2 commented:
Ті чё бля

ср, 11 січ. 2023 р., 01:13 користувач Yehor Akunishnikov <
***@***.***> пише:

> Assigned #3 <https://github.com/yehor-akunishnikov/pablo-html-css/pull/3>
> to @rzotr2 <https://github.com/rzotr2>.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/yehor-akunishnikov/pablo-html-css/pull/3#event-8196681239>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASVMEMCBU4NQ2E3ORPSBXS3WRXUILANCNFSM6AAAAAATXLF4VA>
> .
> You are receiving this because you were assigned.Message ID:
> ***@***.***
> com>
>

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+
+
+
+//.bi-chevron-right {
```
Remove it

yehor-akunishnikov commented on src/index.html:
```diff
@@ -220,6 +221,47 @@
+                        </div>
+                        <div class="user-card-review__text">
+                                <span>
+                                    «Замовила капкейки як подарунок на новий рік. Робити замовлення було легко і приємно, а результат дико порадував, друзі були в захваті. І оформлення, і на смак капкейки були чудові. Обов'язково замовлятиму ще)) Вже будую плани, щоб спробувати всі начинки. Вони чудові!) Навіть їсти було шкода, така краса!)
```
It should be a paragraph, not span

yehor-akunishnikov commented on src/index.html:
```diff
@@ -220,6 +221,47 @@
+                    </div>
+                    <div class="user-card-info">
+                        <div class="user-card-info__image">
+                            <img src="/dist/img/reviews/reviews.jpg" alt="1" style="width: 380px;height: 380px; border-radius: 50%">
```
Why you made styles for that image via inline styles?
It must be moved to the section's stylesheet.

yehor-akunishnikov commented on src/index.html:
```diff
@@ -220,6 +221,47 @@
+                        <div class="user-card-info__image">
+                            <img src="/dist/img/reviews/reviews.jpg" alt="1" style="width: 380px;height: 380px; border-radius: 50%">
+                        </div>
+                        <div class="user-card-info__title">
```
What is the purpose of adding that containers?
In fact it could be p element, or even h5 or h6, and it will work good without any additive containers.

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+.reviews__section {
+  background: #081130;
```
Could be moved to SCSS variable, to make it reusable

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+.reviews__title {
+  margin-bottom: 50px;
+
+  color: #ffffff;
```
Also should be a variable

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+
+  margin: 0 0 24px 0;
+
+  color: #2E363E;
```
Also good candidate to become a variable

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+  }
+}
+
+.user-card-review__text {
```
Rules from that statement could be moved to upper `.user-card-review__title`

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+  color: #2E363E;
+  font-size: 21px;
+  font-weight: 500;
+  line-height: 143%;
```
Why do you need that strange percent values for line-height?

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+
+.user-card-info__subtitle {
+  font-size: 16px;
+  line-height: 178%;
```
Also strange line-height in percents. Looks confusing.

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+}
+
+.reviews-arrows__container {
+  margin-top: 70px;
```
Try to keep margins direction from up to down, that margin should be set in the upper element, and be a margin-bottom.

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+  margin-right: 24px;
+
+  color: #2E363E;
+  font-weight: bold;
```
That `font-weight: bold` makes no sense, because it will not work with icons

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+  font-weight: bold;
+
+  &:last-child {
+    margin: 0;
```
Could be simplified using `:not(:last-child)`.
Also please replace margin with margin-right.

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+  font-size: 21px;
+  font-weight: 500;
+  line-height: 143%;
+  margin-top: 30px;
```
Try to keep margins direction from up to down, that margin should be set in the upper element, and be a margin-bottom.

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+  color: #2E363E;
+  font-weight: 500;
+  font-size: 30px;
+  line-height: 143%;
```
Strange percent line-height, please find a better solution.

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+  &:before {
+    content: "❝";
+
+    margin: 0 0 24px 0;
```
margin-bottom: 24px;

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+}
+
+.user-card-review__link {
+  font-size: 14px;
```
Please use font-size in rem here and all over the section.

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+
+.user-card-review {
+  width: 60%;
+  margin: 0 54px 0 0;
```
margin-right: 54px;

yehor-akunishnikov commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,123 @@
+  display: flex;
+  flex-direction: column;
+
+  margin: 0 0 24px 0;
```
margin-bottom: 24px;

yehor-akunishnikov commented on src/index.html:
```diff
@@ -220,6 +221,47 @@
+                <div class="user-card">
+                    <div class="user-card-review">
+                        <div class="user-card-review__title">
+                            <span>"Результат дико порадував, друзі були в захваті"</span>
```
Should be h4.

rzotr2 replied:
in DIV?

yehor-akunishnikov commented on src/index.html:
```diff
@@ -220,6 +221,47 @@
+    <section class="reviews__section">
+        <div class="container">
+            <div class="reviews__title title">
+                <span>Почитайте відгуки задоволених клієнтів</span>
```
Should be h2 or h3.

rzotr2 replied:
in DIV?

yehor-akunishnikov reviewed (commented):
There are some problems with semantics, and code style, also you forgot about scss variables. Please check all comments and fix that issues.

rzotr2 committed:
[Reviews Section]

Fixed issues described at comments on GitHub:
- Improved semantics (div --> h1, h2, h3 etc.).
- Changed PX values to REM values.
- Fixed other issues.

WannaAlwaysBe commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,127 @@
+$white-background: #ffffff;
+$main-background: #081130;
```
Plese create variables.scss and store all variables in this file.

WannaAlwaysBe commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,127 @@
+
+  background-color: $white-background;
+  box-shadow: 2px 8px 16px rgba(194, 198, 208, 0.06);
+  border: 2px solid #E2EAF2;
```
Color should be a variable.

rzotr2 replied:
![image](https://user-images.githubusercontent.com/78299696/212350237-f2c099ae-fa50-4ce5-946b-288a940c3f66.png)

WannaAlwaysBe commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,127 @@
+
+  margin: 30px 0 24px 0;
+
+  color: #2E363E;
```
Should be a variable.

WannaAlwaysBe commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,127 @@
+
+  font-size: 0.875rem;
+  line-height: 21px;
+  color: #4A94FF;
```
Should be a variable.

WannaAlwaysBe commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,127 @@
+  line-height: 21px;
+  color: #4A94FF;
+
+  border-bottom: 1.5px dashed #4A94FF;
```
Color should be a variable.

WannaAlwaysBe commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,127 @@
+}
+
+.user-card-info__title {
+  color: #2E363E;
```
Should be a variable.

WannaAlwaysBe commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,127 @@
+.user-card-info__subtitle {
+  font-size: 1rem;
+  line-height: 178%;
+  color: #A6AEB6;
```
Should be a variable.

WannaAlwaysBe commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,127 @@
+    width: 2em;
+    height: 2em;
+
+    background-color: #FED84C;
```
Should be a variable.

rzotr2 replied:
![image](https://user-images.githubusercontent.com/78299696/212350373-95ba9a43-4f68-44ab-9dd9-af49469a0c1e.png)

WannaAlwaysBe commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,127 @@
+    font-size: 2rem;
+    margin-right: 24px;
+
+    color: #2E363E;
```
Should be a variable.

rzotr2 committed:
[Reviews Section]

Fixed issues described at comments on GitHub:
- Improved semantics (div --> h1, h2, h3 etc.).
- Changed PX values to REM values.
- Created variables.scss.
- Main colors are in variables.scss.

WannaAlwaysBe commented on src/scss/reviews.scss:
```diff
@@ -0,0 +1,129 @@
+@import "variables";
```
This import should be in index.sccc

rzotr2 replied:
![image](https://user-images.githubusercontent.com/78299696/212132979-aa8ea9cb-3add-4540-ad29-577e918959dc.png)

rzotr2 replied:
when removing import variables.scss in file where variables are used.

WannaAlwaysBe replied:
This is normal behavior. You shouldn't import variables every time

rzotr2 committed:
[Reviews Section]

Fixed issues:
- '@import 'varibles.scss'' removed from the file.

rzotr2 committed:
[Orders Section]

Fixed issues:
- '@import 'varibles.scss'' added to index.scss.

rzotr2 commented:
Allo allo
